DesignSafe-CI/DS-User-Guide
# feat: new Jupyter hierarchy (prep for new Jupyter doc)

## Overview

1. Change JupyterHub hierarchy to start with Jupyter.
2. Add a new _sample_ Jupyter doc.

> [!NOTE]
> I removed content from new _sample_ Jupyter doc, so this PR can be merged to `main` with minimal visible change, so https://github.com/DesignSafe-CI/DS-User-Guide/pull/21 can target `main` branch, so I can step out of the review/approval process.

## Related

No ticket.

## Changes

- **added** new doc in `jupyterhub/`
- **renamed** "JupyterHub" to "Jupyter"

> [!NOTE]
> I do not change directory nor file name, because that will change URL, which will require support for redirects, which I would prefer to implement via #18.

## Testing

1. Serve site.
2. See "Jupyter" in nav.
4. Open "Jupyter" docs.
5. Verify new sample doc, "JupyterSomething" is available.

## UI

| Sample Page with Content (6f48caf) | Sample Page no Content (cd6abb1) |
| - | - |
| <img width="1194" alt="new-jupyter-doc-and-hierarchy" src="https://github.com/DesignSafe-CI/DS-User-Guide/assets/62723358/7d0cda95-001b-422f-ac39-a18167a31316"> | <img width="1024" alt="no new jupyter page visible" src="https://github.com/DesignSafe-CI/DS-User-Guide/assets/62723358/214c64d0-9993-43a3-bb8b-beac495c8864"> |
